src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -130,7 +140,7 @@ private Path resolvePath(Path path, String errorMessage) throws IOException {
      * @return true if the processed ended succesfully, false otherwise
      * @throws IOException in case default path for Substrate dependencies can't be created
      */
-    private boolean setupDependencies() throws IOException {
+    public boolean setupDependencies() throws IOException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'setupDependencies()' has an NPath complexity of 249480](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=4992870)